### PR TITLE
chore: rename 'chatbot' helm chart to 'ircbot'

### PR DESCRIPTION
**Pull request by lemeurherve**
Ref: https://github.com/jenkins-infra/helm-charts/issues/74

### CODE REVIEW

File: updatecli/updatecli.d/charts/chatbot.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-title: "Bump Account App Helm Chart Version"
+title: "Bump `ircbot` helm chart version"
```
Comment (timja) [inline updatecli/updatecli.d/charts/chatbot.yaml:1]: rename updatecli file?

Comment (lemeurherve) [reply updatecli/updatecli.d/charts/chatbot.yaml:1]: Good catch!

**Review by timja — APPROVED** [review]

**Comment by lemeurherve** [comment]
Need the other related PRs to be merged first.

**Comment by lemeurherve** [comment]
updatecli fails because of the changes introduced in this PR, merging.

**Pull request merged**